Automattic/wp-calypso
# Closing account: Clear store after

## Problem

Social First Signup: closing the user account before finishing onboarding results in the user being redirected to /start/domains when attempting to sign up for a new account.
The problem was that the Redux store was not cleared after closing the account, and so the `user-social` step status remained as completed.

## Testing
1. While in Incognito, navigate to `[Live Link]/start`.
2. Click on "Continue with Email".
3. Enter a valid email address, or some gibberish address, doesn't matter. 
5. You should now be in `/start/domains`. Open a new tab and navigate to the live link again.
6. Perform standard Calypso procedure to close the account.
7. Once the account is closed, navigate again to `[Live Link]/start`
8. You should see the Social First signup page.


### Refer to the linked issue to see a video of the testing.

Fixes https://github.com/Automattic/wp-calypso/issues/82645
